recipes/fipy/meta.yaml

```diff
@@ -42,7 +42,6 @@ test:
 about:
   home: http://www.ctcms.nist.gov/fipy
   license: NIST public domain
-  license_file: LICENSE.rst
```

Comment:
You still need the `licence_file` entry, you just don't need a copy of the file itself if it's in the tarball as conda will automatically look in the source tarball for it.

Reply:
Yeah, I realized that right after I posted. Fixed now.